.github/actions/get-changed-gha-items/Get-GhaItemsFromFileList.ps1

```diff
@@ -4,18 +4,38 @@ param(
 
 # Filter actions based on files in action directory.
 [array]$actionFiles = $FileIncludeList | Where-Object -FilterScript {
-    (Get-Item $PSitem).Directory.GetFiles('action.yml').Count -gt 0 -or
-    (Get-Item $PSitem).Directory.GetFiles('action.yaml').Count -gt 0
+    try
```

Comment:
The `try` and `catch` block along with the `action.yml/yaml` check could be removed now, if we only checking the files in the `.github` directory.

Reply:
Updated. Pipeline run is red again since there are unexpected refs to `@dev`.